packages/react-interpolate-components/.gitignore

```diff
@@ -0,0 +1,4 @@
+node_modules
```

Comment:
Ah right... good call. I had copied this code from the separated interpolate-components repository and hadn't thought about the need to change the `.gitignore`.